script/check-st2-ok

```diff
@@ -51,9 +52,31 @@ if [ -x "${ST2_PATH}" ]; then
     echo "First time starting this machine up?"
     echo "Visit https://${ST2_IP}/setup to configure StackStorm"
     echo "Otherwise, head to https://${ST2_FQDN} to access the WebUI"
-    echo ""
-    echo "If you would like to use the CLI interface, please log out and"
-    echo "login to your session again. This is necessary to properly source"
-    echo "StackStorm ENV variables"
+    echo
+    echo "If you would like to use the CLI interface, you will need"
+    echo "to ensure that StackStorm environment variables are properly"
+    echo "set. You can do this by logging out and logging back in, or"
+    echo "running the command:"
+    echo
+    echo ". /etc/profile.d/st2.sh"
+    echo
+    echo "Thanks for installing StackStorm! Come visit us in our Slack Channel"
+    echo "and tell us how it's going. We'd love to hear from you!"
+    echo "http://stackstorm.com/community-signup"
+}
+
+# Only run the check for StackStorm in the event that it actually exists
+if [ -f /usr/bin/st2 ]; then
+  sudo st2 run core.local date &> /dev/null
+  ACTIONEXIT=$?
+  if [ ! "${ACTIONEXIT}" = 0 ]; then
+    retry_message
+    exit 1 # Failure with the self-check
+  else
+    ok_message
+    exit 0
   fi
+else
+  retry_message
+  exit 2 # StackStorm is not installed on the system
```

Comment:
and then returning 'oh noes' and non-zero like it's actually a big deal.

@jfryman sleepy, eh? =) So is it a big deal or is it not?